server/routes/api-v1/userRoutes.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import * as authController from '../../controllers/authController';
+
+const userRouter = express.Router();
+
+userRouter.get('/login', authController.login);
```

Comment:
I think login should be POST method

Reply:
No it shouldn't. This is how OAuth works